CI failure STRICT_JS doesn't work with MODULARIZE or EXPORT_ES6

## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=218479
Build error leg or test failing: Build / browser-wasm linux Release LibraryTests / Build product
Pull request: https://github.com/dotnet/runtime/pull/83952
<!-- Error message template  -->
## Error Message

```
emcc(0,0): error : STRICT_JS doesn't work with MODULARIZE or EXPORT_ES6
❌src/mono/wasm/wasm.proj(445,5): error MSB3073: The command "bash -c 'source /usr/local/emscripten/emsdk/emsdk_env.sh 2>&1 && cmake --build . --config Release '" exited with code 2.
````

```json
{
  "ErrorMessage": "STRICT_JS doesn't work with MODULARIZE or EXPORT_ES6",
  "BuildRetry": false,
  "ErrorPattern": "STRICT_JS doesn't work with MODULARIZE or EXPORT_ES6",
  "ExcludeConsoleLog": false
}
```
https://github.com/dotnet/runtime/pull/83952/checks?check_run_id=12310423547

<!--Known issue error report start -->
### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[219647](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219647)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219647/logs/40)|dotnet/runtime#83589|
|[219639](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219639)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219639/logs/72)|dotnet/runtime#83657|
|[219631](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219631)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219631/logs/28)|dotnet/runtime#83670|
|[219518](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219518)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219518/logs/73)|dotnet/runtime#83776|
|[219587](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219587)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219587/logs/40)|dotnet/runtime#84013|
|[219305](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219305)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219305/logs/73)|dotnet/runtime#84009|
|[219303](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219303)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219303/logs/2343)|dotnet/runtime#84009|
|[219516](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219516)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219516/logs/723)|dotnet/runtime#83776|
|[219514](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219514)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219514/logs/40)|dotnet/runtime#79147|
|[219512](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219512)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219512/logs/398)|dotnet/runtime#79147|
|[219501](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219501)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219501/logs/971)||
|[219483](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219483)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219483/logs/40)|dotnet/runtime#82832|
|[219480](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219480)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219480/logs/875)|dotnet/runtime#82832|
|[219474](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219474)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219474/logs/797)|dotnet/runtime#83976|
|[219476](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219476)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219476/logs/40)|dotnet/runtime#83976|
|[219456](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219456)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219456/logs/58)||
|[219414](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219414)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219414/logs/28)|dotnet/runtime#84011|
|[219412](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219412)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219412/logs/560)|dotnet/runtime#84011|
|[219351](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219351)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219351/logs/28)|dotnet/runtime#82981|
|[219349](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219349)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219349/logs/558)|dotnet/runtime#82981|
|[219313](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219313)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219313/logs/40)|dotnet/runtime#83707|
|[219311](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219311)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219311/logs/570)|dotnet/runtime#83707|
|[219293](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219293)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219293/logs/406)|dotnet/runtime#83948|
|[219282](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219282)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219282/logs/40)|dotnet/runtime#84003|
|[219280](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219280)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219280/logs/396)|dotnet/runtime#84003|
|[219268](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219268)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219268/logs/28)|dotnet/runtime#84004|
|[219266](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219266)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219266/logs/567)|dotnet/runtime#84004|
|[219257](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219257)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219257/logs/40)|dotnet/runtime#83992|
|[219255](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219255)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219255/logs/429)|dotnet/runtime#83992|
|[219251](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219251)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219251/logs/813)|dotnet/runtime#83903|
|[219253](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219253)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219253/logs/40)|dotnet/runtime#83903|
|[219200](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219200)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219200/logs/458)|dotnet/runtime#83755|
|[219202](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219202)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219202/logs/40)|dotnet/runtime#83755|
|[219185](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219185)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219185/logs/467)|dotnet/runtime#83995|
|[219169](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219169)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219169/logs/40)|dotnet/runtime#83554|
|[219167](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219167)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219167/logs/816)|dotnet/runtime#83554|
|[219119](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219119)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219119/logs/844)|dotnet/runtime#83945|
|[219121](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219121)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219121/logs/40)|dotnet/runtime#83945|
|[219073](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219073)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219073/logs/46)|dotnet/runtime#83707|
|[219071](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219071)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219071/logs/588)|dotnet/runtime#83707|
|[219047](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219047)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219047/logs/442)|dotnet/runtime#83911|
|[219044](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219044)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219044/logs/40)|dotnet/runtime#83903|
|[219022](https://dev.azure.com/dnceng-public/public/_build/results?buildId=219022)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/219022/logs/436)|dotnet/runtime#84000|
|[218834](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218834)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218834/logs/73)|dotnet/runtime#83996|
|[218830](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218830)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218830/logs/1346)|dotnet/runtime#83996|
|[218963](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218963)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218963/logs/40)|dotnet/runtime#83892|
|[218960](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218960)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218960/logs/400)|dotnet/runtime#83892|
|[218939](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218939)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218939/logs/441)|dotnet/runtime#83827|
|[218942](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218942)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218942/logs/40)|dotnet/runtime#83827|
|[218899](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218899)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218899/logs/28)|dotnet/runtime#83903|
|[218869](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218869)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218869/logs/408)|dotnet/runtime#79147|
|[218871](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218871)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218871/logs/40)|dotnet/runtime#79147|
|[218874](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218874)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218874/logs/305)||
|[218860](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218860)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218860/logs/40)|dotnet/runtime#81617|
|[218857](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218857)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218857/logs/807)|dotnet/runtime#81617|
|[218847](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218847)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218847/logs/601)|dotnet/runtime#83707|
|[218849](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218849)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218849/logs/46)|dotnet/runtime#83707|
|[218802](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218802)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218802/logs/422)|dotnet/runtime#83995|
|[218799](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218799)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218799/logs/40)|dotnet/runtime#83827|
|[218795](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218795)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218795/logs/40)|dotnet/runtime#83827|
|[218790](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218790)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218790/logs/40)|dotnet/runtime#83838|
|[218788](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218788)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218788/logs/728)|dotnet/runtime#83838|
|[218775](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218775)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218775/logs/916)||
|[218769](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218769)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218769/logs/383)|dotnet/runtime#83992|
|[218771](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218771)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218771/logs/40)|dotnet/runtime#83992|
|[218749](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218749)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218749/logs/711)|dotnet/runtime#83554|
|[218751](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218751)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218751/logs/40)|dotnet/runtime#83554|
|[218736](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218736)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218736/logs/40)|dotnet/runtime#83990|
|[218733](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218733)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218733/logs/440)|dotnet/runtime#83990|
|[218731](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218731)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218731/logs/58)||
|[218720](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218720)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218720/logs/836)|dotnet/runtime#83982|
|[218722](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218722)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218722/logs/40)|dotnet/runtime#83982|
|[218712](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218712)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218712/logs/421)|dotnet/runtime#83911|
|[218654](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218654)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218654/logs/438)|dotnet/runtime#82031|
|[218619](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218619)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218619/logs/395)|dotnet/runtime#83948|
|[218611](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218611)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218611/logs/40)|dotnet/runtime#83554|
|[218546](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218546)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218546/logs/40)|dotnet/runtime#79147|
|[218489](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218489)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218489/logs/40)|dotnet/runtime#83844|
|[218487](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218487)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218487/logs/374)|dotnet/runtime#83844|
|[218479](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218479)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218479/logs/462)|dotnet/runtime#83952|
|[218481](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218481)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218481/logs/40)|dotnet/runtime#83952|
|[218465](https://dev.azure.com/dnceng-public/public/_build/results?buildId=218465)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/218465/logs/493)|dotnet/runtime#83911|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|82|82|82|
<!--Known issue error report end -->

--- a/eng/pipelines/common/templates/pipeline-with-resources.yml
+++ b/eng/pipelines/common/templates/pipeline-with-resources.yml
@@ -60,10 +60,10 @@ resources:
         ROOTFS_DIR: /crossrootfs/ppc64le
 
     - container: browser_wasm
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly-net8
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly-net8-20230322221728-80fdceb
 
     - container: wasi_wasm
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-webassembly-net8
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-webassembly-net8-20230322221804-80fdceb
 
     - container: freebsd_x64
       image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12

--- a/eng/pipelines/libraries/helix-queues-setup.yml
+++ b/eng/pipelines/libraries/helix-queues-setup.yml
@@ -201,6 +201,6 @@ jobs:
 
     # Browser WebAssembly windows
     - ${{ if in(parameters.platform, 'browser_wasm_win', 'wasi_wasm_win') }}:
-      - (Windows.Amd64.Server2022.Open)windows.amd64.server2022.open@mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-ltsc2022-helix-webassembly-net8
+      - (Windows.Amd64.Server2022.Open)windows.amd64.server2022.open@mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-ltsc2022-helix-webassembly-net8-20230319084205-80fdceb
 
     ${{ insert }}: ${{ parameters.jobParameters }}

